src/components/Editor/Preview.js

```diff
@@ -99,7 +99,7 @@ class Preview extends Component {
   renderObjectInspector(root) {
     const { loadObjectProperties, loadedObjects } = this.props;
 
-    const getObjectProperties = id => loadedObjects.get(id);
+    const getObjectProperties = id => loadedObjects[id];
```

Comment:
this makes me think we might want to use a JS `Map`... We'd have a slightly nicer API